slam-libraries/viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -51,11 +53,13 @@ static const int samplingFactor = 1;
 // Conversion to number of bytes used in colored PCD encoding
 static const int pixelBytetoPCDByte = 16 / 4;
 // Quaternion to rotate axes to the XZ plane
-static const Eigen::Quaterniond pcdRotation(0.7071068, 0.7071068, 0, 0);
+static const Eigen::Quaterniond pcdRotation(0.7071068, -0.7071068, 0, 0);
```

Comment:
Why is this changing?

Reply:
the change here was to flip the map in the viewer on cartographer's side, to get the map in the expected orientation